src/components/MenuItem.js

```diff
@@ -224,14 +237,17 @@ const MenuItem = React.forwardRef((props, ref) => {
                                         <View style={[styles.flexRow, styles.alignItemsCenter]}>
                                             {Boolean(props.title) && Boolean(props.shouldRenderAsHTML) && <RenderHTML html={convertToLTR(props.title)} />}
 
-                                            {Boolean(props.title) && !props.shouldRenderAsHTML && (
+                                            {Boolean(html.length) && !props.shouldRenderAsHTML && <RenderHTML html={`<comment>${html}</comment>`} />}
+
```

Comment:
This looks incorrect - if shouldRenderAsHTML is false, we use renderHTML?

Reply:
@thienlnam if you see the code for updating `html` then you would be able to understand why we render `html` when shouldRenderAsHTML is false.

Reply:
As they both have somewhat different usecases

Reply:
Ah I see, that is confusing - can we add another shouldParseTitle check in this conditional?

Reply:
Sure